app/src/main/res/values/strings.xml

```diff
@@ -437,4 +437,5 @@ Without it, you won’t be able to recover your Safe and money.</string>
     <string name="channel_description_wallect_connect_sessions">Updates for WalletConnect sessions</string>
     <string name="notification_new_transaction_request">New transaction request to review</string>
     <string name="select_safe_to_connect">Select a Safe to connect</string>
+    <string name="search_select_tokens"><![CDATA[Search & select tokens…]]></string>
```

Comment:
what is CDATA needed here for?

Reply:
because of the & .... this was generated by android studio